package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "club-view",
+  "version": "1.0.0",
+  "description": "ClubView Main Repo",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:frontend": "cd frontend && npm run start",
+    "start:backend": "cd backend && npm run start",
+    "build:frontend": "cd frontend && npm run build",
+    "build:backend": "cd backend && npm run tsc",
+    "install:frontend": "cd frontend && npm install",
+    "install:backend": "cd backend && npm install",
+    "postinstall": "npm run install:frontend && npm run install:backend",
+    "quiet-lint": "npx eslint . --ext .js --ext .jsx --ext .ts --ext .tsx --fix --quiet",
```

Comment:
We don't need the `npx` part since both `eslint` and `prettier` are dev dependencies